.github/workflows/pr-language-labeler.yml

```
@@ -0,0 +1,110 @@
+name: PR Language Labeler
+
+on:
+  pull_request:
+    types: [ opened, ready_for_review, synchronize ]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  labeling-languages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Create package.json
+        run: echo '{}' > package.json
+
+      - name: Install dependencies
+        run: npm install @octokit/rest node-fetch
+
+      - name: Detect languages and add labels
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

nit: 다른 secret은 몰라도 GitHub 토큰은 `github` 컨텍스트에서 읽는 것이 좀 더 일반적인 것 같습니다.

```suggestion
          GITHUB_TOKEN: ${{ github.token }}
```

---

오 이 부분은 몰랐던 내용이네요 공유해주셔서 감사합니다!